Classes/DDFileLogger.m

```diff
@@ -1044,18 +1096,24 @@ - (void)logMessage:(DDLogMessage *)logMessage {
     }
 
     if (message) {
-        if ((!isFormatted || _automaticallyAppendNewlineForCustomFormatters) &&
-            (![message hasSuffix:@"\n"])) {
```

Comment:
Like above, I think we're fine with expecting `\n` to be the newline character here.